tools/flux/tool-data/huggingface.loc.sample

```diff
@@ -0,0 +1,18 @@
+
+#This is a sample file distributed with Galaxy that is used to define huggingface
+#models, using 7 columns tab separated
+#(longer whitespace are TAB characters):
+#
+#The entries are as follows:
+#
+#<unique_build_id>	<display_name>	<pipeline_tag>	<domain>	<free_tag>	<version>	<folder_base_path>
+#
+#pipeline_tag (see this URL: https://huggingface.co/models?pipeline_tag=text-to-image they call it that way)
+#domain (defined by the Galaxy community, e.g. image / sequence / text)
+#free_tag (not idea yet, but could be freely used by an admin to specify more filter options)
+#version (if available)
+#Your huggingface.loc file should include the huggingface cached model
+#folder you have stored.  For example:
+#
+#black-forest-labs/FLUX.1-dev	FLUX.1 [dev] is an open-weight, guidance-distilled model for non-commercial applications.	text-to-image	image	flux	1	/path/to/huggingface/that-contains/hub/models--black-forest-labs--FLUX.1-dev
```

Comment:
@beatrizserrano @anuprulez @kostrykin maybe you are interested in this.

We are trying to come up with a potential Galax data table for huggingface. So that Galaxy instance admin can download models deposit them locally and register them in such data tables.

Different tools can then depend on that table and filter them according to a tiny bit of metadata.